packages/gatsby/src/utils/flags.ts

```diff
@@ -179,7 +179,21 @@ const activeFlags: Array<IFlag> = [
     telemetryId: `LmdbStore`,
     experimental: true,
     umbrellaIssue: `https://gatsby.dev/lmdb-feedback`,
-    description: `Store nodes in a persistent embedded database (vs in-memory). Lowers peak memory usage.`,
+    description: `Store nodes in a persistent embedded database (vs in-memory). Lowers peak memory usage. Requires Node v14.10 or above.`,
+    testFitness: (): fitnessEnum => {
+      const [major, minor] = process.versions.node.split(`.`)
+      return (Number(major) === 14 && Number(minor) >= 10) || Number(major) > 14
+    },
+  },
+  {
+    name: `PARALLEL_QUERY_RUNNING`,
+    env: `GATSBY_EXPERIMENTAL_PARALLEL_QUERY_RUNNING`,
+    command: `build`,
+    telemetryId: `PQR`,
+    experimental: true,
+    umbrellaIssue: `https://gatsby.dev/pqr-feedback`,
+    description: `Parallelize running page queries in order to better sature all available cores. Improves time it takes to run queries during gatsby build. Requires Node v14.10 or above.`,
```

Comment:
Should it be `sature` -> `saturate` ?

```suggestion
    description: `Parallelize running page queries in order to better saturate all available cores. Improves time it takes to run queries during gatsby build. Requires Node v14.10 or above.`,
```